electron/game.ts

```diff
@@ -83,13 +74,35 @@ export class Game {
     })
   }
 
-  private updateWindow(window: Window, logFileName: string): void {
-    const title = window.title()
-    if (POE_TITLES.includes(title) || POE_ALTERNATIVE_TITLES.some((x) => title.startsWith(x))) {
-      this.window = window
-      this.active = true
-      this.bounds = window.bounds()
-      this.gameLogListener.setLogFilePath(path.join(path.parse(window.path).dir, "logs", logFileName))
+  private updateWindow(window: Window): void {
+
+    const windowPath = path.parse((window.path || '').toLowerCase())
+
+    if (POE_NAMES.includes(windowPath.base)) {
```

Comment:
Since you're using `path.parse` anyway, the `POE_NAMES` could probably benefit from an optimization too by removing all `.exe` names, and using `windowPath.name` instead of the `.base`